roman.py

```diff
@@ -33,21 +33,21 @@ def get_roman_digit(number, base):
     if quotient >= 5:
         # do something including the half base
         if quotient == 9:
-            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)])) 
+            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)])) 
```

Comment:
The first thing you mentioned can be resolved with
```
from __future__ import print_function

print("stuff", end="")
```

but i think i second way is better still. have this function build a string
and return it. let `main` do the printing

On Wed, Apr 13, 2016 at 1:48 PM, yiwork <notifications@github.com> wrote:

> In roman.py
> <https://github.com/yiwork/codesample/pull/1#discussion_r59593195>:
>
> > @@ -33,21 +33,21 @@ def get_roman_digit(number, base):
> >      if quotient >= 5:
> >          # do something including the half base
> >          if quotient == 9:
> > -            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)]))
> > +            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)]))
>
> So the reason why I did sys.stdout.write is because I wanted to print
> each character as the script goes along. print's new line character got
> in the way.
>
> Perhaps a better way to do this is to use a string and keep concatenating
> each character, and print once the whole number is converted.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/yiwork/codesample/pull/1/files/719196351561909d289d232ef0a20ae10f733239#r59593195>
>